### PR TITLE
Disable PasswordlessMySQLIT 

### PR DESCRIPTION
**Pull request by backwind1233**
# Description
Now the pipeline `java - spring - tests` is blocked because of PasswordlessMySQLIT, it can't be fixed in a few days, so disable the IT, use other issues and PRs to track and fix the issue.

## Followups
- [ ] https://github.com/Azure/azure-sdk-for-java/issues/31476

### CODE REVIEW

File: sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/jdbc/mysql/PasswordlessMySQLIT.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+// @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
```
Comment (chenrujun) [inline sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/jdbc/mysql/PasswordlessMySQLIT.java:18]: How about adding `@DisableTest` here?

File: sdk/spring/tests.yml [hunk 2]
```diff
@@ -12,9 +12,8 @@ stages:
           SubscriptionConfiguration: $(sub-config-gov-test-resources)
         China:
           SubscriptionConfiguration: $(sub-config-cn-test-resources)
-          Location: 'chinanorth'
+          Location: 'chinanorth3'
```
Comment (chenrujun) [inline sdk/spring/tests.yml:15]: Is this used for test? How about reverting unnecessary change?

Comment (backwind1233) [reply sdk/spring/tests.yml:15]: It's another story, 
it's `chinanorth3` in main branch, but changed to `chinanorth` when merging passwordless feature to main.

File: sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/jdbc/mysql/PasswordlessMySQLIT.java (hunk 1 above)
Comment (backwind1233) [reply sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/jdbc/mysql/PasswordlessMySQLIT.java:18]: It's ok, I will have a try.

**Review by saragluna — APPROVED** [review]
